src/PyDynamic/model_estimation/fit_filter.py

```diff
@@ -411,12 +426,15 @@ def _compute_x(
     return x
 
 
-def _compute_and_print_rms(residuals_real_imag: np.ndarray) -> np.ndarray:
+def _compute_and_print_rms(
+    residuals_real_imag: np.ndarray, suppress_print=False
+) -> np.ndarray:
     rms = np.sqrt(np.sum(residuals_real_imag**2) / (len(residuals_real_imag) // 2))
-    print(
-        f"{_get_first_public_caller()}: Calculation of filter coefficients finished. "
-        f"Final rms error = {rms}"
-    )
+    if not suppress_print:
+        print(
+            f"{_get_first_public_caller()}: Calculation of filter coefficients finished. "
+            f"Final rms error = {rms}"
+        )
```

Comment:
I guess we should streamline this implementation. It does not seem very obvious and clear to me, that we have to introduce another parameter here, when we could simply split the function.

Reply:
meanwhile resolved by splitting both tasks in  separate functions